portal/server/routes/contract/loan/index.js

```diff
@@ -355,12 +332,12 @@ router.get('/contract/:_id/loan/:loanId/check-your-answers', provide([LOAN]), as
   });
 });
 
-router.get('/contract/:_id/loan/:loanId/issue-facility', provide([LOAN, DEAL]), async (req, res) => {
+router.get('/contract/:_id/loan/:loanId/issue-facility', provide([LOAN, DEAL]), validateRole({ role: ['maker'] }), async (req, res) => {
```

Comment:
That's a very good point, I'll make this change everywhere

Reply:
I've also made this change for `validateBank` (i.e., ensuring that it is after `validateRole` in every instance), as this can also make API requests to `portal-api`

Reply:
Nice spot!